src/openMVG/cameras/Camera_Intrinsics.hpp

```diff
@@ -128,7 +128,7 @@ struct IntrinsicBase
 };
 
 /// Return the angle (degree) between two bearing vector rays
-static double AngleBetweenRay(
+static inline double AngleBetweenRay(
```

Comment:
No we should remove the static. It doesn't make sense to use that in header files.
Here the inline was required to avoid compilation errors. But yes, we need to remove all "static" methods.